docs/sources/tempo/shared/metamonitoring.md

```diff
@@ -0,0 +1,201 @@
+---
+headless: true
+description: Shared file for metamonitoring GET and Tempo.
+labels:
+  products:
+    - enterprise
+    - oss
+---
+
+[//]: # 'This file documents the best practices for monitoring for Tempo.'
+[//]: # 'This shared file is included in these locations:'
+[//]: # '/tempo/docs/sources/tempo/operations/monitor/set-up-monitoring.md'
+[//]: #
+[//]: # 'If you make changes to this file, verify that the meaning and content are not changed in any place where the file is included.'
+[//]: # 'Any links should be fully qualified and not relative: /docs/grafana/ instead of ../grafana/.'
+
+
+Metamonitoring for Tempo is handled by the [Grafana Kubernetes Helm chart](https://github.com/grafana/k8s-monitoring-helm) (>=v2.1). Using Grafana Alloy to collect metrics and logs. Metamonitoring can be used with both microservices and single binary deployments of Tempo.
+
```

Comment:
At the end of this procedure, we instruct people to log into Grafana Cloud to check their stats. What if they are using Grafana open source and not Cloud? 

This is one difference for GET vs Tempo. GET requires Cloud while Tempo users can specify Grafana OSS or Cloud. 

Also, when we're in the Tempo docs, we try to minimize any emphasis on Cloud.

Reply:
I have removed any specific references to cloud I added. I did rework the URL help text in the values.yaml so double check what you like there. In Alloy we found people left off things all the time. IE only put something like `https://loki.grafana.net`

Reply:
I've often wondered how many times people forget to update the key fields in the yaml files and then wonder why it doesn't work.